### ISSUE TITLE
V2 Docs need updating

### ISSUE THREAD

**Issue by TimGeyssens** [TimGeyssens/UIOMatic#52]
Since there are quite a few differences that need to be documented and of course all the new bits

**Comment by TimGeyssens** [comment]
Docs have been updated http://uiomatic.readthedocs.io/en/stable/

**Comment by mattbrailsford** [comment]
The ListViewFilter attribute docs are still TODO.

**Comment by mattbrailsford** [comment]
Still to do
- [ ] Change logo for one on a transparent background
- [ ] ListViewFilter docs
- [ ] DateTimeUtc view
- [ ] Label config options (formatting?)
- [ ] Expand on UIOMaticRepository, include example, mention the strongly typed AbstractUIOMaticRepository
- [ ] Actions docs
- [ ] Consistant casing in doc menu
- [ ] http://uiomatic.readthedocs.io/en/stable/02.listview/ has a code snippet not showing as code block
- [ ] In the example on the home page, use UIOMatic not UIOMaticAttribute
- [ ] Do we need to document the list view fields like we do the editor fields?
- [ ] http://uiomatic.readthedocs.io/en/stable/04.CustomEditorViews/#custom-field-editor-views update the angular view to be like what we are now doing in our dropdown views

**Comment by TimGeyssens** [comment]
Thanks for the review :) could you take a look at the logo since I don't have it with transparent bg

**Comment by mattbrailsford** [comment]
Yup, no problem. I can handle any you are unsure of the details for, I just wanted to dump down what I've noticed.

**Comment by TimGeyssens** [comment]
The Repo example would be ace since i'haven't done that yet the rest I can take :)

**Comment by mattbrailsford** [comment]
No problem. I can handle that :)

**Comment by TimGeyssens** [comment]
Woohoo so close now :) , any chance you can share a snippet on how to use the ListViewFilters since I'm a bit unsure about that

**Comment by mattbrailsford** [comment]
Sure thing :)

**Comment by TimGeyssens** [comment]
Haha just seen the sorry marc comment :)

**Comment by TimGeyssens** [comment]
Also added a small note on dashboards, I think docs are complete now woohoo!

**Comment by mattbrailsford** [comment]
Yup, I think it's pretty much there :)

Go UI-O-Matic team! 🤖

**Comment by TimGeyssens** [comment]
Yihaa! Can't wait to try out the repo stuff you added, so much goodness in this release, thanks so much! ❤️
